src/victory-util/style.js

```diff
@@ -1,3 +1,30 @@
+import { pick } from "lodash";
+
+/**
+ * Acceptable CSS/SVG style attributes
+ * https://react-cn.github.io/react/docs/tags-and-attributes.html#svg-attributes
+ */
+const styleWhitelist = [
+  "angle", "clipPath", "cx", "cy", "d", "dx", "dy", "fill", "fillOpacity", "fontFamily",
+  "fontSize", "fx", "fy", "gradientTransform", "gradientUnits", "height", "markerEnd",
+  "markerMid", "markerStart", "offset", "opacity", "patternContentUnits", "patternUnits",
+  "points", "preserveAspectRatio", "r", "rx", "ry", "spreadMethod", "stopColor", "stopOpacity",
+  "stroke", "strokeDasharray", "strokeLinecap", "strokeOpacity", "strokeWidth", "textAnchor",
+  "transform", "version", "verticalAnchor", "viewBox", "width", "x1", "x2", "x", "xlinkActuate",
+  "xlinkArcrole", "xlinkHref", "xlinkRole", "xlinkShow", "xlinkTitle", "xlinkType", "xmlBase",
+  "xmlLang", "xmlSpace", "y1", "y2", "y"
+];
+
+/**
+ * Given an object with CSS/SVG style attributes, return a new object containing
+ * only keys that are also on our SVG style whitelist.
+ * @param {Object} data An object of user-input style attributes.
+ * @returns {Object} An object containing only valid style data.
+ */
+export const sanitizeStyleProps = function (data) {
```

Comment:
I think this export is unnecessary, since you're exporting it with your `export default` below

Reply:
I just want to add a quick note here - I was pairing with Kylie and noticed that we cannot use named imports from default exports. I'm unfamiliar with the project semantics, but is this intentional?

ie, removing `export` here but then using the named `import { sanitizeStyleProps } from './style'` will cause `sanitizeStyleProps` to be undefined.

It looks the the way to avoid that is to import the entire module and path to the method like this https://github.com/FormidableLabs/victory-core/blob/73534d40536957e1a681f97ec776686371084895/src/victory-label/victory-label.js#L8

Not sure if this is a bug, or intentional but wanted to alert you, or @ryan-roemer to it.